src/App.tsx

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const searchFilter = (movie: Movie) => {
+    const { title, description } = movie;
+    const searchLowerCase = search.toLowerCase().trim();
+
+    return title.toLowerCase().includes(searchLowerCase)
+      || description.toLowerCase().includes(searchLowerCase);
+  };
+
+  const filteredMovies = moviesFromServer.filter(searchFilter);
+
+  const handleChange = (event: {
+    target: { value: React.SetStateAction<string>; }; }) => {
+    setSearch(event.target.value);
```

Comment:
it's hard to read
Check eventType into `onChange` on `input`
```suggestion
  const handleChange = (event: eventType }) => {
    setSearch(event.target.value);
```